production/docker/docker-compose.yaml

```diff
@@ -115,19 +113,19 @@ services:
       - loki
 
   loki-read:
-    image: grafana/loki:2.8.2
+    image: *lokiImage
     volumes:
       - ./config:/etc/loki/
       - ./rules:/loki/rules:ro
     ports:
       - "3100"
       - "7946"
       # uncomment to use interactive debugging
-      # - "40000-40002:40000" # # makes the replicas available on ports 40000, 40001, 40002
-      #cap_add:
-      #  - SYS_PTRACE
-      #security_opt:
-      #  - apparmor=unconfined
+      # - "40000-40002:40000" # makes the replicas available on ports 40000, 40001, 40002
+    # cap_add:
```

Comment:
Weird indentation here

Reply:
yeah, it's messy because `cap_add` is on the same level as `ports`. I reorganized the file so the indentation is cleaner/more obvious.